docs/native/src/differences.rst

```diff
@@ -1,3 +1,16 @@
+..  _differences:
+
+Differences between |Office iOS| and |wac|
+==========================================
+
+Office for iOS has two WOPI implementation requirements which are not relevant to Office Online:
+ * **Containers**: Instead of using the ecosystem to traverse the WOPI space, Office for iOS uses containers. A container represents a folder on the cloud storage provider, and a Root Container is a top-level folder. 
```

Comment:
I'd avoid describing containers and ecosystem here. Instead, I'd simply say that partners need to implement all WOPI operations on the ecosystem and containers endpoints, plus the bootstrapper. There's already a todo on the endpoints page to give more details on what the ecosystem and containers endpoints are for, and that or the key concepts section is a better place to give context for what they are. The intent of this page is just to highlight the differences.